OxfordBodleian/Juel-Jensen/BDLaethg34.xml

```diff
@@ -160,25 +118,671 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
                         
                     </physDesc>
                     
-                <!--    <history>
+                        <history>
                         <origin>
-                            <origDate when="" evidence=""/>
+                            <ref target="#p1"/> can very probably be dated by the mention of <persName ref="PRS5616IyasuI"/>
```

Comment:
That's a good point. The question is whether we want an overall history element for the entire manuscript to summarise the history of all parts and present the entire picture? This is what I did here (without much thinking), but it certainly is somewhat redundant.
Here, the owner is attested in two parts, so this information at least would make sense in a history element pertaining to the entire manuscript.
For now, I will split the information according to units and note this point in our agenda, so that we can discuss it later.

Reply:
yes, I think this is a good solution for the moment. I am not against either way (one single history element or as many history elements as the codicological units), we can discuss in the next meeting about the implications for our printed catalogue.